### ISSUE TITLE
Accessibility spot check review for redesigned VA.gov home page

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#43789]
### What must be true in order for you to consider this epic complete? 

*Take into consideration Accessibility/QA needs as well as Product, Technical, and Design requirements.*

[ ] Accessibility tickets associated with this epic should be closed.

## How to configure this issue
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `Console-Services`, `tools-fe`,`content-ia`, `service-design`, `vsa-*`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)

**Comment by mmiddaugh** [comment]
Attached [SCREENREADER]: Alt attributes need to be more descriptive #43898 to Public Websites post-MVP homepage epic and will now close this epic
